### DISCUSSION TITLE
Fermata Auto didnt show the folders

### DISCUSSION THREAD

**Discussion by Alexviana10** [AndreyPavlenko/Fermata#185] [General]
Is there some issue in fermata auto? My list dissapeared and the folder is empty.

**Comment by AndreyPavlenko** [comment]
Have you tried to restart the app?

**Comment by Alexviana10** [comment]
I reinstalled it.

The folder appears empty.

Em ter., 2 de ago. de 2022 18:02, Andrey ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
Have you added a folder after reinstallation?

**Comment by Alexviana10** [comment]
Andrey i reinstalled and inserted the link (m3u) in the Location.
And the file and folder shows zero.

Em ter., 2 de ago. de 2022 18:28, Andrey ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
I thought you are talking about file folders.
Could you share the link so I could check?

**Reply by AndreyPavlenko** [reply]
Try changing the user agent in settings, for example, to set Google as the user agent. I'm getting an empty response from the server with the default agent.

**Comment by Alexviana10** [comment]
Just write "Google"?

Em ter., 2 de ago. de 2022 20:30, Andrey ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
Right.
Btw, why do you add this link to folders? I'd recommend using the TV addon.

**Comment by Alexviana10** [comment]
Im lost... hahaha...

Em ter., 2 de ago. de 2022 20:37, Andrey ***@***.***>
escreveu:

**Comment by Alexviana10** [comment]
Andrey,

Any idea?

Maybe the link has a problem?
But I used it in another app and works. Iptv player live.

Em ter., 2 de ago. de 2022 20:39, Alexandre Viana ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
As I said yesterday, the server returns an empty list if if does not recognize the user agent. Changing the agent string should solve the problem.

**Comment by Alexviana10** [comment]
The same thing

Em qua., 3 de ago. de 2022 17:41, Andrey ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
For some reason, the server returns nothing. You can check it with the curl utility:

$ curl -i 'http://4f.rs/get.php?username=xxx&password=xxx&type=m3u_plus&output=ts'
HTTP/1.1 200 OK
Server: nginx
Date: Wed, 03 Aug 2022 21:32:00 GMT
Content-Type: text/html; charset=UTF-8
Connection: close
Access-Control-Allow-Origin: *

**Comment by Alexviana10** [comment]
How can I use the add-ons?

Em qua., 3 de ago. de 2022 18:34, Andrey ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
This link does not work with the TV addon either. You can download the playlist and add it as a file.

**Comment by Alexviana10** [comment]
As a Content?

Em qua., 3 de ago. de 2022 19:44, Andrey ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
No, either playlist or TV source, but, instead of URL, specify the file path.

**Comment by Alexviana10** [comment]
I really dont know what happened.... it was working very well... im
disapointed

Em qua., 3 de ago. de 2022 19:59, Andrey ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
I assume, something has changed on the server. Just click on the link on your phone and download the playlist, then add the downloaded file to the app.

**Comment by Alexviana10** [comment]
Works

Em qua., 3 de ago. de 2022 20:36, Andrey ***@***.***>
escreveu:

**Comment by Alexviana10** [comment]
Andrey, yesterday I dowloaded the list and everything was ok... this
morning the list dissapeared...

Can you explain that? Hahaha

Em qua., 3 de ago. de 2022 21:00, Alexandre Viana ***@***.***>
escreveu:

**Reply by AndreyPavlenko** [reply]
Is the file path still valid? Didn't you move or delete the file?
